### PR TITLE
:sparkles: Reconcile with latest rulesets

### PR DESCRIPTION
**Pull request by pranavgaikwad**
Reconcile 
- https://github.com/konveyor/rulesets/pull/23
- https://github.com/konveyor/rulesets/pull/27

### CODE REVIEW

File: resources/rulesets.yaml [hunk 1]
```diff
@@ -184,3 +184,4 @@ items:
         - 10667615-e067-45b4-a925-563a79ea0949
         - 1b2a0eef-511c-4c07-a36a-d301b15d8207
       checksum: 3668a741fc5025557a192b7ab5712237a718aefac99c19248d62fb88151c0c5c
+root: ./resources
```
Comment (pranavgaikwad) [inline resources/rulesets.yaml:187]: I am unsure about this change in rulesets.yaml file. Can someone tell if this is correct or not?

Comment (mansam) [reply resources/rulesets.yaml:187]: @pranavgaikwad This doesn't belong here, what is the intention of adding this?

**Comment by mansam** [comment]
Did you run `make ruleset-patch`?

File: resources/rulesets.yaml (hunk 1 above)
Comment (pranavgaikwad) [reply resources/rulesets.yaml:187]: I just ran `go run ./cmd/ruleset/` and it added that.

**Pull request closed without merging**